src/main/java/com/unb/tracker/repository/CourseRepository.java

```diff
@@ -15,4 +15,7 @@
 
     @Query(value = "SELECT * FROM course WHERE course.name LIKE CONCAT('%',:pname,'%') OR course.section LIKE CONCAT('%',:pname,'%') LIMIT 10", nativeQuery = true)
     public List<Course> findByPartialName(@Param("pname")String partialName);
+
+    public Course findById(Long courseId);
```

Comment:
Is this not the same method as `findOne(Long id)` which is implemented automatically?